packages/input/src/input.vue

```diff
@@ -252,6 +272,11 @@
         if (this.$slots[pendant]) {
           return { transform: `translateX(${place === 'suf' ? '-' : ''}${this.$el.querySelector(`.el-input-group__${pendant}`).offsetWidth}px)` };
         }
+      },
+      clear() {
+        this.$emit('input', '');
+        this.setCurrentValue('');
+        this.focus();
```

Comment:
Hi, everything looks good to me, except that after clearing the input, the `change` event should trigger. Here we should add
```JS
this.$emit('change', '');
```